feat: Add FSM state backfill functionality (fb-fit-710-3)

yyassi-heartex opened the pull request:
This pull request introduces a new visual style for the `Badge` component by adding a `success` variant. This makes it easier to display success messages or statuses with a consistent appearance across the application.

Badge component update:

* Added a `success` variant to the `badgeVariants` configuration in `badge.tsx`, using positive background and content colors for improved visual indication of successful actions.

yyassi-heartex committed:
feat: Add FSM state backfill functionality (fb-fit-710-3)

bmartel commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19765423497)

bmartel committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19765423497

bmartel committed:
Merge branch 'develop' into 'fb-fit-710-fsm-state-backfill'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19765423497
